site/docs/src/json/messengers/generic-message-request.json

```diff
@@ -0,0 +1,5 @@
+{
+  "phoneNumber": "3035551212",
+  "textMessage": "Two Factor Code: 111111",
+  "type": "SMS"
```

Comment:
Just FYI, reviewing some of this code as the result of this doc update. Opened this GH issue to add a few more things to make integration easier. 
https://github.com/FusionAuth/fusionauth-issues/issues/1574